resources/prometheus/prometheus-rules.yaml

```
@@ -756,3 +756,16 @@ cluster autoscaler. Limits can be adjusted by modifying the cluster autoscaler c
 the cluster autoscaler. This is calculated by summing the memory capacity for all nodes in the cluster and comparing that number against the maximum memory bytes value set
 for the cluster autoscaler. Limits can be adjusted by modifying the cluster autoscaler configuration."
             sop_url: "https://gitlab.cee.redhat.com/stackrox/acs-cloud-service/runbooks/-/blob/master/sops/dp-041-modify-cluster-autoscaler.md"
+        - alert: ClusterAuditSELinuxViolations
+          expr: |
+            max_over_time(selinux_denials_sample_count[5m]) >= 1
+          for: 1m
```

```suggestion
            selinux_denials_sample_count > 0
```
Wouldn't this be the easier condition with essentially the same behavior (alert if at any time there are violations found)?